NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -8,7 +8,6 @@ namespace NitroxClient.MonoBehaviours.Gui.MainMenu
     public class JoinServer : MonoBehaviour
     {
         public string serverIp = "";
-        string username = "username";
         bool showingUsername = false;
```

Comment:
Did you mean I should reimplement it? As backup or as option to enter the name before joining?
I am happy to remove this ugly and buggy input field with a nice solution.

Reply:
You've moved the username input to the menu, so I guess (and by looking at the code) that means that this boolean is doing nothing anymore since no username input field is shown anymore in this 'join' menu. So, I guess you can remove it.

Reply:
Yes, I misunderstood you. I fix this :)